Dockerfile

```diff
@@ -97,6 +97,7 @@ RUN \
     curl \
     ffmpeg \
     file \
+    imagemagick \
```

Comment:
We do not want `imagemagick` in our container image. We should rather set `MASTODON_USE_LIBVIPS` in the `Dockerfile`, so VIPS is used with this image, as the proper vips version is installed